### DISCUSSION TITLE
Does ccache support cross compiler for aarch64?

### DISCUSSION THREAD

**Discussion by t0ny-peng** [ccache/ccache#783] [Questions & answers] [answered]
We are doing cross compiling for aarch64 platform and `ccache` will be a great help. Before I dive into it, just wonder if it supports cross compilers like `aarch64-linux-gnu-g++`?

Much obliged.

**Comment by AlexanderLanin** [comment, marked as answer]
No reason it would not work.
Have a look at https://ccache.dev/manual/4.1.html and ask here if anything remains unclear. It's not that much diving.
In case you are using CMake you can reuse this: https://github.com/ccache/ccache/blob/master/cmake/UseCcache.cmake

**Reply by t0ny-peng** [reply]
@AlexanderLanin Thanks for the information. I'll try to integrate it into our build system. Might be a little bit tricky.
